pages/platform/ai/gi_07_manage_registry/guide.en-gb.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Use & manage existing and new image registries
+slug: use-manage-registries
+excerpt: Learn how to use and manage your public and private image registries
+section: General information
+order: 109
+updated: 2023-05-02
+---
+
+**Last updated 2nd May, 2023.**
+
+## Objective
+
+This guide is intended to help you **use and manage your public and private registries** through ovhai command line interface (AI CLI), API and OVHcloud control panel. This will help you when working with **OVHcloud AI Tools**, especially with **AI Training** and **AI Deploy**.
+
+## Requirements
+
+- A **Public Cloud** project
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
+
+## Instructions
+
+This documentation is divided into the following parts:
+- Manage existing registries through
```

Comment:
Please, check if the rendering is OK (not sure if it is ok with the double `-` and indentations)

Reply:
Indentations will work. See an example on https://help.ovhcloud.com/csm/en-gb-nutanix-metro-availability?id=kb_article_view&sysparm_article=KB0045186#instructions

Reply:
Perfect, thanks